research/src/pages/select/select.proposal.mdx

```diff
@@ -116,11 +116,60 @@ children:
   Should this apply to the entire anatomy or solely the &lt;option&gt; and &lt;optgroup&gt; element
 </p>
 
+## Events <a class="link" href="#events" id="events"></a>
+
+### select
+
+| Event    | Behavior                                                                                         | Impacts          |
+| -------- | ------------------------------------------------------------------------------------------------ | ---------------- |
+| `change` | Updates the textContent property of the selected-value slot with the value property text content | textContent prop |
+
+### part button
+
+| Event          | Behavior                           | Impacts                                                                  |
+| -------------- | ---------------------------------- | ------------------------------------------------------------------------ |
+| `click`        | Toggles the open state on the host | [open](#open-state) state                                                |
+| `click`        | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `pointerup`    | Toggles the open state on the host | [open](#open-state) state                                                |
+| `pointerup`    | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `keyup(space)` | Toggles the open state on the host | [open](#open-state) state                                                |
+| `keyup(space)` | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `keyup(enter)` | Toggles the open state on the host | [open](#open-state) state                                                |
+| `keyup(enter)` | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+
+### part listbox
+
+| Event             | Behavior                                                                                                               | Impacts                                                                  |
+| ----------------- | ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| `keyup(down key)` | Moves focus to the next `<option>` in the listbox and sets the                                                         | focus                                                                    |
```

Comment:
Similar vein to the above, but today OSX moves focus on *keydown*.

Reply:
is this for all `keyup` events I listed or solely focus movement? It does seem on Windows it to be `keydown` for moving focus. That said, on selection with pointer it is on `pointerdown` but the state of open is not changed until `pointerup`.

Reply:
One important difference with `keydown` vs `keyup` is that holding down the arrow key and having it spin quickly through the options will only work with `keydown.`

Reply:
Yea this seems to be for all keyboard events. I can't test the selection keys because selection is happening on focus, but the reset of them are happening on keydown and not up.

Reply:
Thanks, for now then I'll move them to `keydown` until someone can prove otherwise.